app/assets/stylesheets/pages/_events_show.scss

```diff
@@ -0,0 +1,55 @@
+
+.logo {
+text-align: center;
+font-weight: 300;
+font-size: 35px;
+}
+
+.title1 {
```

Comment:
be specific about title. Just numbering 1,2,3,4,5 does not indicate anything. Big? Small? something like this

Reply:
ok. I will rename. I just did it as the first title on the page. the second title on the page.